packages/block-library/src/navigation-link/test/edit.js

```diff
@@ -0,0 +1,506 @@
+/**
+ * Internal dependencies
+ */
+import { updateNavigationLinkBlockAttributes } from '../edit';
+
+describe( 'edit', () => {
+	describe( 'updateNavigationLinkBlockAttributes', () => {
+		// data shapes are linked to fetchLinkSuggestions from
```

Comment:
We need to document these data shapes with typedefs. Can we import typedefs from another package to use in another package?

PS: when can we have TypeScript support! 🙏